### ISSUE TITLE
Fix notices

### ISSUE THREAD

**Issue by iandunn** [WordPress/wporg-mu-plugins#59]
- [x] `Trying to get property 'id' of non-object in wp-content/plugins/gutenberg/lib/global-styles.php:21`. Happens on pages like https://br.wordpress.org/gutenberg/. I think it started after some of the commits in #49 or #46 , or maybe from network-activating the Gutenberg plugin on Rosetta sites.
- [x] `Undefined offset: 2366 in global-header-footer/blocks.php:584`. Source: GET https://it.wordpress.org/wp-json/global-header-footer/v1/header - maybe disable for most sites? see #45 
- [x] `Undefined index: url in /global-header-footer/blocks.php:765`. Source: GET https://it.wordpress.org/wp-json/global-header-footer/v1/header  - maybe disable for most sites? see #45  
- [x] `Undefined index: title in global-header-footer/header.php:50`. Source: GET https://it.wordpress.org/wp-json/global-header-footer/v1/header. Also happens for `url` and `type`  - maybe disable for most sites? see #45

**Comment by dd32** [comment]
> Trying to get property 'id' of non-object in wp-content/plugins/gutenberg/lib/global-styles.php:21.

Caused by the `gutenberg` theme including some admin-related files required to make the gutenberg editor render on the front-end, making `get_current_screen()` available. Gutenberg plugin assumes `get_current_screen()` will return a specific value, it doesn't. Not a bug we caused.

> Undefined offset: xxx in global-header-footer/blocks.php:584
> Undefined index: url/title in /global-header-footer/blocks.php:765

On it.wordpress.org, these were caused by the Rosetta menu having children of the Downloads page, which was removed from the menu, but the children were not. I likely caused that during the bulk transition to hard-coded rosetta menus.

On af.wordpress.org, these were caused by the Rosetta menu being slightly corrupt, `post_parent` was 0, but `menu_item_parent` was non-zero pointing to a non-menu-ID.

I think these can be avoided by changing `normalize_rosetta_items()`'s `} else {` to `} elseif ( ! empty( $parent_indices[ $item->menu_item_parent ] ) ) {`.
